packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -426,13 +428,17 @@ export class WorkspaceCommandContribution implements CommandContribution {
         }
     }
 
-    protected async addFolderToWorkspace(uri: URI | undefined): Promise<void> {
-        if (uri) {
+    protected async addFolderToWorkspace(...uris: URI[]): Promise<void> {
```

Comment:
`undefined` is no longer a valid input type for this method. Does this qualify as a breaking change?

Reply:
Yes, I think it does, even though we weren't availing ourselves of that option previously.

Reply:
Right, I also don't think anyone is really affected by this change in any way. Should we still add it as a breaking change to the changelog?